bench/cardano-profile/app/cardano-profile.hs

```diff
@@ -20,8 +23,9 @@ data Cli =
     Names
   | NamesCloudNoEra
   | All
-  | ByName String
+  | ByName Bool String              -- 1st arg: True for pretty-printing
```

Comment:
To avoid boolean blindness you could define a sum type for this instead.

Reply:
✅